xapian-core/configure.ac

```diff
@@ -504,6 +515,7 @@ AC_SEARCH_LIBS([timer_create], [rt],
 #endif]])],
 	[AC_MSG_RESULT([yes])
 	XAPIAN_LIBS="$LIBS $XAPIAN_LIBS"
+	XAPIAN_CMAKE_LIBS=";rt$XAPIAN_CMAKE_LIBS"
```

Comment:
This parallel assembling of the same information but in a different format feels really brittle - someone adding a check for a new library needs to remember to also add it for cmake, or else we only discover much later when someone tries to use cmake+static libraries.  There's also nothing to catch if they get the cmake part wrong.

Can't we just do something like:

```
XAPIAN_CMAKE_LIBS=`echo " $XAPIAN_LIBS"|sed 's/ -l/;/g;s/ //g'`
```

Reply:
You're right.
I copy pasted your regex.